Documentation/ApiOverview/CachingFramework/Developer/Index.rst

```diff
@@ -1,143 +1,165 @@
+        protected function getCachedValue()
+        {
+            $cacheIdentifier = /* ... logic to determine the cache identifier ... */;
 
-           // If $entry is false, it hasn't been cached. Calculate the value and store it in the cache:
-           if (($value = $this->cache->get($cacheIdentifier)) === false) {
-               $value = /* ... Logic to calculate value ... */;
-               $tags = /* ... Tags for the cache entry ... */
-               $lifetime = /* ... Calculate/Define cache entry lifetime ... */
+            // If $entry is false, it hasn't been cached. Calculate the value and store it in the cache:
+            if (($value = $this->cache->get($cacheIdentifier)) === false) {
+                $value = /* ... Logic to calculate value ... */;
+                $tags = /* ... Tags for the cache entry ... */
+                $lifetime = /* ... Calculate/Define cache entry lifetime ... */
 
-               // Save value in cache
-               $this->cache->set($cacheIdentifier, $value, $tags, $lifetime);
-           }
+                // Save value in cache
+                $this->cache->set($cacheIdentifier, $value, $tags, $lifetime);
+            }
 
-           return $value;
-       }
+            return $value;
+        }
 
-.. tip::
+..  tip::
+    It is not needed to call :php:`$this->cache->has()` before accessing cache
+    entries with :php:`$this->cache->get()` as the latter returns :php:`false`
+    if no entry exists.
 
-   It isn't needed to call :code:`has()` before accessing cache entries with :code:`get()`
-   as the latter returns :code:`false` if no entry exists.
+Since the auto-wiring feature of the dependency injection container cannot
+detect which cache configuration should be used for the :php:`$cache` argument
+of :php:`MyClass`, the :ref:`container service configuration
+<configure-dependency-injection-in-extensions>` needs to be extended:
 
-Since the auto-wiring feature of the dependency injection container cannot detect
-which cache configuration should be used for the :php:`$cache` argument of :php:`MyClass`,
-the :ref:`container service configuration <configure-dependency-injection-in-extensions>`
-needs to be extended:
-
-.. code-block:: yaml
+..  code-block:: yaml
+    :caption: EXT:some_extension/Configuration/Services.yaml
 
     services:
+      # other configurations
+
       Vendor\SomeExtension\MyClass:
         arguments:
           $cache: '@cache.myext_mycache'
 
-Here `@cache.myext_mycache` refers to the cache service we defined above. This setup allows you
-to freely inject the very same cache into any class.
+Here :yaml:`@cache.myext_mycache` refers to the cache service we defined above.
+This setup allows you to freely inject the very same cache into any class.
+
+..  note::
+    After changes in the :file:`Services.yaml` file flush the cache via
+    :guilabel:`Admin Tools > Maintenance` or the :ref:`CLI command
+    <symfony-console-commands>` `cache:flush`.
```

Comment:
Put the full command here as convenience to copy as code-block? At least I love doing that ;)

Reply:
Hm, then we have to add also the legacy path with tabs which makes it a little bit big just for clearing the cache.

Reply:
We should just make a template for clearing the cache once and then include it wherever needed?

Reply:
can be done in a follow up though